### PR TITLE
Fix make report

### LINKED ISSUE #12: Single model HTML report path is hard-wired in makeReport.py
Just saw that the HTML report path is hard-wired, in makeReport.py

I think it should be set in the paths script.

The default location is fine though:
```
    reportdir = folder('reports/' + jobID)
```

### PR DESCRIPTION
**Pull request by ledm**
Fixes the report maker so that it can work from any location. To do this, I've added the repository base directory to paths as paths.bgcval2_repo.

Closes #12 

Change log:

- Changed `makeReport.py` to `bgcval2_make_report.py` (start of addressing #18.)
- Changed several hard-wired relative paths to absolute paths relative to `paths.bgcval2_repo`

### CODE REVIEW

File: bgcval2/analysis_compare.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def timeseries_compare(jobs,
 
     # filter paths dict into an object that's usable below
     paths = paths_setter(paths_dict)
-
+
```
Comment (valeriupredoi) [inline bgcval2/analysis_compare.py:176]:
```suggestion
```

**Comment by ledm** [comment]
Before we accept this PR, I need to go though all the places that say "./data/" and change it to `os.path.join(paths.bgcval2_repo, 'bgcval2/data'`

File: bgcval2/html5/html5Tools.py [hunk 2]
```diff
@@ -33,6 +33,14 @@
 from ..bgcvaltools.pftnames import getLongName
 
 
+
+def get_this_file(html5_file):
+    """
+    gets the correct to a specific file in html5.
```
Comment (valeriupredoi) [inline bgcval2/html5/html5Tools.py:39]:
```suggestion
    gets the correct path to a specific file in html5.
```

**Review by valeriupredoi — APPROVED** [review]
looks great, man!

**Comment by ledm** [comment]
> Before we accept this PR, I need to go though all the places that say "./data/" and change it to `os.path.join(paths.bgcval2_repo, 'bgcval2/data'`

Okay, this is done now, I think.

**Comment by valeriupredoi** [comment]
cool! I'll add those lil bits from my review, and when the tests have finished I be merging

**Pull request merged**